evalml/model_understanding/graphs.py

```diff
@@ -100,29 +100,36 @@ def graph_confusion_matrix(y_true, y_pred, normalize_method='true', title_additi
         plotly.Figure representing the confusion matrix plot generated
     """
     _go = import_or_raise("plotly.graph_objects", error_msg="Cannot find dependency plotly.graph_objects")
+    _ff = import_or_raise("plotly.figure_factory", error_msg="Cannot find dependency plotly.figure_factory")
     if jupyter_check():
         import_or_raise("ipywidgets", warning=True)
 
     conf_mat = confusion_matrix(y_true, y_pred, normalize_method=None)
     conf_mat_normalized = confusion_matrix(y_true, y_pred, normalize_method=normalize_method or 'true')
-    labels = conf_mat.columns
+    labels = conf_mat.columns.tolist()
 
     title = 'Confusion matrix{}{}'.format(
         '' if title_addition is None else (' ' + title_addition),
         '' if normalize_method is None else (', normalized using method "' + normalize_method + '"'))
     z_data, custom_data = (conf_mat, conf_mat_normalized) if normalize_method is None else (conf_mat_normalized, conf_mat)
+    z_data = z_data.to_numpy()
+    z_text = [["{:.3f}".format(y) for y in x] for x in z_data]
```

Comment:
Is this supposed to only show 3 sig figs? The example in the docs is still a lot of sig figs 🤔 
![image](https://user-images.githubusercontent.com/5422235/103585925-0467f880-4eb2-11eb-9762-4dab07643a29.png)

Reply:
@angela97lin the docs are still building. Not sure why they're taking so long, but this is what I see. 
![image](https://user-images.githubusercontent.com/22552445/103586440-17c79380-4eb3-11eb-8294-7351fa7654ed.png)
They should update when the build finishes though!

Reply:
@angela97lin @freddyaboulton I updated the description to include an image of what the graph looks like on a local doc build. Since BTD is currently failing, I'll hold off on merging until we can make sure the actual docs will look as expected.

Reply:
Doc build is passing and can be seen [here](https://evalml.alteryx.com/en/bc_1601_confusion/user_guide/model_understanding.html#Confusion-Matrix)